app/controllers/api/v1x0/portfolio_items_controller.rb

```diff
@@ -36,6 +41,13 @@ def destroy
         head :no_content
       end
 
+      def copy
+        svc = Catalog::CopyPortfolioItem.new(portfolio_copy_params)
+        render :json => svc.process.new_portfolio_item
+      rescue ActiveRecord::RecordNotFound => e
```

Comment:
@lindgrenj6 This seems redundant since this be caught at a higher level

Reply:
Sure - I'll remove this catch and handle the error I was really meaning to catch later, I need to throw a 422 if the user passes in a bad portfolio_id in the body (since we're doing 422 instead of 404).

Reply:
Ended up just defining a new exception, `Catalog::InvalidParameter`, which I throw if the user-defined Portfolio doesn't exist. I then handle it with a 422.